### ISSUE TITLE
Option to delete original file after postprocessing while using --split-chapter

### ISSUE THREAD

**Issue by jordanbecketmoore** [yt-dlp/yt-dlp#4686]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.08.14** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

Suppose we want to download a channel or playlist that contains individual videos of full albums. We attempt to download the all of its content using,
`yt-dlp -x --split-chapter <channel/playlist url>`
yt-dlp will extract audio and then split the chapters into individual tracks as desired, but the original full-album file from which the individuals tracks were produced remains. If we are using yt-dlp for music archival purposes (as I am), this can end up taking up a lot of unnecessary storage and can be quite a pain to clean up after the fact. It would be nice to have an option, `--no-keep-unsplit` for example, that will remove the unsplit file after splitting is complete. 

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by pukkandan** [comment]
You can just use `--exec rm` or `--exec del` depending on your OS

**Comment by candrapersada** [comment]
how to use options similar to `"%(playlist)s/%(title)s.%(ext)s"` in `--split-chapter`?

**Comment by pukkandan** [comment]
I don't understand the question

**Comment by candrapersada** [comment]
what I mean is how to use an option similar to `Download YouTube playlist videos in separate directory indexed by video order in a playlist`

**Comment by pukkandan** [comment]
and that is related to OP's question how?

**Comment by candrapersada** [comment]
after download complete --split-chapter does not save in %(playlist)s/ folder like `-o "%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s"`

**Comment by kqvanity** [comment]
@pukkandan That removes the original media file whether it contains chapters or not. Can I conditionally make it so that It only removes the originally file if it expanded to chapters (in a separate dir) ?

**Comment by kqvanity** [comment]
I've tried something along the lines of

```
yt-dlp --split-chapters --parse-metadata "%(chapters|)l:(?P<has_chapters>.)" --match-filters 'has_chapters' --exec after_filter:'ls %(filename)q' -S 'height:720' --write-sub --write-auto-sub --sub-langs en --convert-subs srt --paths Software\ Talks --output 'chapter:%(title)s/%(section_title)s-%(section_number)s.%(ext)s' --batch-file new.txt --download-archive ids.txt
```

but it skip downloading altogether.

**Comment by bashonly** [comment]
Yeah there is no perfect solution currently, which is why I've reopened this
